tests/common.rs

```diff
@@ -0,0 +1,47 @@
+use lazy_static::*;
+use std::sync::Arc;
+use tokio::sync::Mutex;
+use unftp_auth_jsonfile::JsonFileAuthenticator;
+use unftp_sbe_fs::ServerExt;
+
+lazy_static! {
+    static ref CONSUMERS: Arc<Mutex<i32>> = Arc::new(Mutex::new(0));
+}
+
+pub async fn run_with_json_auth() {
+    let addr = "127.0.0.1:2121";
+    let server = libunftp::Server::with_fs(std::env::temp_dir())
+        .authenticator(Arc::new(
+            JsonFileAuthenticator::from_json("[{\"username\":\"test\",\"password\":\"test\"}]").unwrap(),
```

Comment:
Would be nice if we can eliminate the dependency on JsonFileAuthenticator.

Reply:
Yes totally, I got rid of it now